frontend/test/pytest/test_autograph.py

```diff
@@ -424,6 +425,57 @@ def func(x):
         assert not np.all(func(0.9) == 1)
 
 
+class TestTransforms:
```

Comment:
Can you add some lit test?

Reply:
You're right, I replaced them with lit tests 👍